includes/discovery/sensors/current/cimc.inc.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Looking at this, it should be possible to do this in yaml, any reason you haven't?